packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -1,86 +1,150 @@
+    userId: 'USERID', //
+    token: 'TOKEN' // string
   },
-  elementToRender, // Container element
+  containerElement, // Container element
   props // Optional, Object contains all optional props for composite, check composite page for more details
 );
 ```
 
+_Check [Chat Composite Basic Example](./?path=/docs/composites-chat-basicexample--basic-example) for details on what additional properties you can apply to the chat composite_
+
+### Code to load the call composite
+
+We start with our necessary arguments (displayName, groupId, userId, token) followed by the html element that will contain our chat composite, and the
+any additional properties for the composite.
+
 ```tsx
 const callAdapter = await callComposite.loadCallComposite(
   //The for composite which allows you control composite externally,
   // This function will be exposed to window.callComposite once you import composite bundle
   {
-    groupId, // string, Provide any GUID to join a group
-    displayName, // string
-    userId, // Object { communicationUserId: string }
-    token // string
+    displayName: 'DISPLAY_NAME', // string
+    groupId: 'GROUPID', // string
+    userId: 'USERID', // string
+    token: 'ACCCESS_TOKEN' // string
   },
-  elementToRender, // container element
+  containerElement, // container element
   props // Optional, Object contains all optional props for composite, check composite page for more details
 );
 ```
 
-Check [Chat Composite Basic Example](./?path=/docs/composites-chat-basicexample--basic-example) and [Call Composite Basic Example](./?path=/docs/composites-call-basicexample--basic-example) to see optional props of composites
+_Check [Call Composite Basic Example](./?path=/docs/composites-call-basicexample--basic-example) for details on what additional properties you can apply to the call composite_
 
-For more details of adapter, check [Adapters for Composites](./?path=/story/composite-adapters--page)
+## Examples
 
-## Example html code
+Now we can take a look at how this all comes together
 
-Check [Example](https://github.com/Azure/communication-ui-library/blob/PublicPreview/1.0.0-beta.3/samples/StaticHtmlComposites/index.html) for more detailed example of using calling/chat composites
+### Chat (index.html) example
 
 ```html
+<meta charset="utf-8" />
 <!-- Replace this url with your own CDN in production environment -->
 <script src="https://github.com/Azure/communication-ui-library/releases/latest/download/chatComposite.js"></script>
 <!-- height need to be set for composite to fit the layout -->
 <div id="chat-container" style="height: 50vh"></div>
 <script type="module">
   const chatAdapter = await chatComposite.loadChatComposite(
     {
-      displayName: displayName,
-      threadId: threadId,
-      endpoint: endpoint,
-      userId: user,
-      token: token
+      displayName: 'DISPLAY_NAME', // string
+      threadId: 'THREAD_ID', // string
+      endpoint: 'ENDPOINT', // string
+      userId: 'USERID', //
```

Comment:
userId: {communicationUserId: 'USERID'}

Reply:
Changing to : { communicationUserId: 'USERID' } // CommunicationUserIdentifier